airbyte-integrations/connectors/source-google-analytics-v4-service-account-only/metadata.yaml

```diff
@@ -22,9 +22,9 @@ data:
   name: Google Analytics (Universal Analytics)
   registries:
     cloud:
-      enabled: true
+      enabled: false
     oss:
-      enabled: true
+      enabled: false
   releaseStage: generally_available
   supportLevel: community
```

Comment:
shouldn't we set the supportLevel to archived [like we did with other connectors we archived](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-appstore-singer/metadata.yaml#L29)?